lib/liquid/standardfilters.rb

```diff
@@ -151,6 +151,23 @@ def sort_natural(input, property = nil)
       end
     end
 
+    # Sort elements of an array in numeric order
+    # provide optional property with which to sort an array of hashes or drops
+    def sort_numeric(input, property = nil)
+      ary = InputIterator.new(input)
+      if property.nil?
+        ary.sort do |a, b|
+          Utils.to_number(a) <=> Utils.to_number(b)
+        end
+      elsif ary.empty? # The next two cases assume a non-empty array.
```

Comment:
isn't this unnecessary (because `ary.first.respond_to?(:[])` will be `false` if `ary` is empty

Reply:
but wouldn't the `else` branch return `nil` instead of `[]` as this branch does?

Reply:
yep I'm an idiot